tests/unit/FmCorrection.spec.ts

```diff
@@ -0,0 +1,390 @@
+import Vue from 'vue'
```

Comment:
For now, this is mostly a copy of the original Correction.spec.ts page (with a few updates).

More to come in future commits...